### PR TITLE
[new][author] Add jschardet library

### PR DESCRIPTION
**Pull request by aadsm**
This refers to #7056 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/aadsm/jschardet.git
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 148
   - Count of watchers: 12
   - Count of forks: 25
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jschardet/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "jschardet",
+  "filename": "jschardet.min.js",
+  "version": "1.4.1",
+  "description": "Character encoding auto-detection in JavaScript (port of python's chardet)",
+  "author": "António Afonso",
+  "license": "LGPL-2.1+",
+  "keywords": ["encoding", "charset"],
```
Comment (PeterDaveHello) [inline ajax/libs/jschardet/package.json]:
```js
  "keywords":
    [
      "encoding",
      "charset"
    ],
```

**Comment by PeterDaveHello** [comment]
Since there is no any tags in the git repo, the auto-update will not work, @aadsm would you please add version tags for it? Thanks.

**Comment by PeterDaveHello** [comment]
Hmmmm ... I saw it has npm auto-update, so I'll respectfully suggest you to transfer to npm auto-update.

https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md#enabling-npmrecommended-or-git-auto-update

**Comment by aadsm** [comment]
Ah, I read it but completely missed this. I'll use git auto-update if that's fine, I don't want to publish this file to npm (it's not needed so it reduces the file size to download).

**Comment by PeterDaveHello** [comment]
@aadsm Oh okay, it will be fine.

**Comment by aadsm** [comment]
@PeterDaveHello Thanks! I've just added the `v1.4.1` tag to the repro and updated the package.json.

**Comment by PeterDaveHello** [comment]
Thanks @aadsm

**Pull request merged**